packages/redot-stringify/index.js

```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line n/file-extension-in-import
+import {stringify} from 'ts-graphviz/ast'
+
+/** @type {import('unified').CompilerFunction<import('ts-graphviz/ast').DotASTNode>} */
+function compiler(tree) {
+  return stringify(tree)
+}
+
+/**
+ * @this {import('unified').Processor}
+ * @type {import('unified').Plugin<void[], import('ts-graphviz/ast').DotASTNode, string>}
```

Comment:
I think this may help

```suggestion
 * @this {import('unified').Processor<import('ts-graphviz/ast').DotASTNode, import('ts-graphviz/ast').DotASTNode, import('ts-graphviz/ast').DotASTNode>}
 * @returns {undefined}
```

Reply:
Thanks! I'll give this a try, but might need to do something like: https://github.com/remarkjs/remark/blob/26dc58a675ac7267c105f0fdb76a82db77f8402a/packages/remark-stringify/index.d.ts#L1-L9 in the end